src/components/LeaderBoard.js

```diff
@@ -163,139 +53,120 @@ function LeaderBoard(props) {
-          className={`${classes.rectangle69} ${classes.colorPalette1} ${classes.clickable}`}
-          align="center"
-          top="100px"
+    <>
+      <Box
+        sx={{
+          flexGrow: 1,
+          borderRadius: 16,
+          borderColor: 'grey.500',
+          marginTop: 5,
+        }}
+      >
+        <Grid
+          container
+          columns={16}
+          justifyContent="center"
+          alignItems="center"
         >
-          <Avatar className={classes.eclipse}>
-            <Avatar
-              className={classes.profileImage}
-              alt="alt name"
-              src={countries[1].flag}
-            />
-          </Avatar>
-          <Box id="inner1" className={classes.rectangle81}>
-            <Box className={classes.planter}>No.2</Box>
-            <Box className={classes.content}>{countries[1].name}</Box>
-            <Box className={classes.rating}>
-              Planted
-              <br></br>
-              {countries[1].planted}
-            </Box>
-          </Box>
-        </Box>
-      </Box>
-      <Box>
-        <Box
-          onClick={() => handleCountryClick(countries[0].id)}
-          className={`${classes.rectangle69} ${classes.colorPalette2} ${classes.clickable}`}
-          align="center"
-          left="220px"
-          top="-100px"
-        >
-          <Avatar className={classes.eclipse}>
-            <Avatar
-              className={classes.profileImage}
-              alt="alt name"
-              src={countries[0].flag}
-            />
-          </Avatar>
-          <Box id="inner2" className={classes.rectangle81}>
-            <Box className={classes.planter}>Planter</Box>
-            <Box className={classes.content}>{countries[0].name}</Box>
-            <Box className={classes.rating}>
-              Planted
-              <br></br>
-              {countries[0].planted}
-            </Box>
-          </Box>
-        </Box>
-      </Box>
-      <Box>
-        <Box
-          onClick={() => handleCountryClick(countries[2].id)}
-          className={`${classes.rectangle69} ${classes.colorPalette3} ${classes.clickable}`}
-          align="center"
-          left="440px"
-          top="-230px"
-          background="linear-gradient(139.31deg, rgba(71, 75, 79, 0.2) 0%, rgba(97, 137, 47, 0.9) 77.54%);"
-        >
-          <Avatar className={classes.eclipse}>
-            <Avatar
-              className={classes.profileImage}
-              alt="alt name"
-              src={countries[2].flag}
-            />
-          </Avatar>
-          <Box id="inner3" className={classes.rectangle81}>
-            <Box className={classes.planter}>Planter</Box>
-            <Box className={classes.content}>{countries[2].name}</Box>
-            <Box className={classes.rating}>
-              Planted
-              <br></br>
-              {countries[2].planted}
-            </Box>
-          </Box>
-        </Box>
+          <Grid item xs={1}>
```

Comment:
Instead of using an empty grid column for spacing, add margins to the Grid's container (in this case the Box on line 57)

Reply:
Thank you